### DISCUSSION TITLE
Use of Reference temperature parameter

### DISCUSSION THREAD

**Discussion by ALMA7867** [pybamm-team/PyBaMM#4645] [General]
Hii everyone, my cell temperature  is going from 35 to 38.7 degree Celsius, ambient temperature is room temperature, So I just want to know which temperature parameters I need to change like there are 3 parameters (Ambient temperature [K],   Initial temperature[K]	Reference temperature [K]).

**Comment by brosaplanella** [comment]
Hi! We have now moved our discussions to [Discourse](https://pybamm.discourse.group/). Please post your question there.
